pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -28,6 +28,8 @@ const (
 	// This should remain in the default role bindings for the NodeAdmin role
 	LegacyMasterKubeletAdminClientUsername = "system:master"
 	MasterKubeletAdminClientUsername       = "system:openshift-node-admin"
+
+	InfraNodeBootstrapServiceAccountName = "system:" + NodeBootstrapRoleName
```

Comment:
this is not the name of the service account... this belongs more in the `controller_policy.go` file than here... it's more akin to the template service broker